src/main/kotlin/com/msg/gauth/domain/image/service/UploadImageService.kt

```diff
@@ -0,0 +1,25 @@
+package com.msg.gauth.domain.image.service
+
+import com.msg.gauth.domain.admin.exception.FileExtensionInvalidException
+import com.msg.gauth.domain.image.presentation.dto.response.UploadImageResDto
+import com.msg.gauth.global.thirdparty.aws.s3.S3Util
+import org.springframework.stereotype.Service
+import org.springframework.web.multipart.MultipartFile
+
+@Service
+class UploadImageService(
+    private val s3Util: S3Util
+) {
+    fun execute(image: MultipartFile): UploadImageResDto {
+        val list = listOf("jpg", "png", "gif")
+
+        val extension = image.originalFilename.toString().split(".")[1]
+
+        if(!list.contains(extension))
```

Comment:
```suggestion
        if(!list.contains(extension.toLowerCase()))
```
로 검사하는거어떤가요? .JPEG 요런식으로도 들어올 수 있으니

Reply:
어 이거 Kotlin의 .any 써보는거 어떤가요

Reply:
lowercase로 변환해서 any로 비교하게 바꿨습니다